customer/tests.py

```diff
@@ -35,10 +36,39 @@ def set_up_customer():
     return customer
 
 
+def set_up_customer_list():
+    """Creates several random CustomUser and related Customer instances in the test DB"""
+
+    user_list = CustomUser.objects.bulk_create(
+        [
+            CustomUser(
+                email=f"{fake.first_name()}.{fake.last_name()}@{fake.domain_name()}"
+            )
+        ]
+    )
```

Comment:
this will create only one user. better approach could be with a `for` loop:

```py
user_list = []
for i in range(10):
  user = CustomUser.objects.create(email=f"{fake.first_name()}.{fake.last_name()}@{fake.domain_name()}")
  user_list.append(user)
```